core/dslmcode/stacks/discuss/sites/all/modules/local_contrib/harmony_core/harmony_core.install

```diff
@@ -634,3 +634,10 @@ function harmony_core_update_7103() {
     variable_set('harmony_core_rebuild_actions', $actions);
   }
 }
+
+/**
+ * Revert cis_displays component in views. This is for caching update. 
+ */
+function harmony_core_update_7104(&$sandbox) {
```

Comment:
well need to do soemthing special to modify this view since its in harmony core which is actually out in contrib and not something we have direct ownership of